tests/ui/uninlined_format_args.stderr

```diff
@@ -774,18 +774,6 @@ LL -     println!("{:w$.p$}", w = width, p = prec);
 LL +     println!("{width:width$.prec$}");
    |
 
-error: variables can be used directly in the `format!` string
-  --> $DIR/uninlined_format_args.rs:125:20
-   |
-LL |     println!("{}", format!("{}", local_i32));
-   |                    ^^^^^^^^^^^^^^^^^^^^^^^^
-   |
-help: change this to
-   |
-LL -     println!("{}", format!("{}", local_i32));
-LL +     println!("{}", format!("{local_i32}"));
-   |
-
```

Comment:
This is a bug with `root_macro_call_first_node`, not sure why but it doesn't pick up that `format!()` as a macro call